coordinator/tests/services/translation/request/prescription.spec.ts

```diff
@@ -432,6 +434,91 @@ describe("createRepeatNumberForMedicationRequests", () => {
   })
 })
 
+describe("extractRepeatNumberHighValue", () => {
+  test("extracts from dispenseRequest", () => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
maybe this is the best way to do these kinds of tests :joy: bit painful to look at tho. could we not do some message/resource cloning from somewhere?

Reply:
Wouldn't be so ugly if the linter allowed me to use `as` :wink:

Reply:
Managed to appease the linter. See what you think now

Reply:
much better :+1: